packages/manager/apps/procedures/src/pages/create/form/Modal/ConfirmModal.tsx

```diff
@@ -0,0 +1,99 @@
+import React, { FunctionComponent } from 'react';
+import {
+  ODS_BUTTON_VARIANT,
+  ODS_ICON_NAME,
+  ODS_ICON_SIZE,
+  ODS_SPINNER_SIZE,
+  ODS_TEXT_COLOR_HUE,
+  ODS_TEXT_LEVEL,
+  ODS_TEXT_SIZE,
+} from '@ovhcloud/ods-components';
+import {
+  OsdsButton,
+  OsdsIcon,
+  OsdsModal,
+  OsdsSpinner,
+  OsdsText,
+} from '@ovhcloud/ods-components/react';
+import { useTranslation } from 'react-i18next';
+import { ODS_THEME_COLOR_INTENT } from '@ovhcloud/ods-common-theming';
+
+type Props = {
+  isPending: boolean;
+  onClose: () => void;
+  onValidate: () => void;
+};
+
+export const ConfirmModal: FunctionComponent<Props> = ({
+  onClose,
+  onValidate,
+  isPending,
+}) => {
+  const { t } = useTranslation('account-disable-2fa');
+
+  return (
+    <OsdsModal
+      dismissible
```

Comment:
Don't we risk to close the confirmation modal while the documents are being uploaded ?
_If so we should handle it properly, as right now I think the user will have access to the form to upload other documents when in reality his/her procedure is being "finalized"_